docs/multiColumnSortTable.md: fix function references.

paulkoegel-wessels opened the pull request:
+ fix function references so they match the initial imported function name (`createMultiSort` -> `createTableMultiSort`).

For exported function name from source/index.js, see:
https://github.com/bvaughn/react-virtualized/blob/de107e81f6d412d3561b169f7fdbfdbe25dd7b00/source/index.js#L21

TrySound commented:
@bvaughn Why did you introduced this inconsistency? `source/Table` and `source/index` should have the same exports since `source/index` currently is not recommended as a good way to import components.

bvaughn commented:
On my mobile so can't really look.

Assume what you're asking about is why I sometimes rename and export with "table" in the name?

This is an old pattern. Doesn't really work well with deep imports and we should probably change it in version 10.

bvaughn commented:
(change it so both versions have the "table" bit at part of the name)

TrySound commented:
@paulkoegel-wessels Could you tweak also source code? Also we still need current version as an alias in both `source/index` and `source/Table` to not introduce breaking changes. You can add `TODO` to remove them later.

bvaughn commented:
Yeah that seems reasonable.

paulkoegel-wessels commented:
@TrySound Sure, I'll take a look. Should I add the `TODO` as a code comment or is there some other place where they are kept in this project?

TrySound commented:
I think it's only in `source/index` and `source/Table/index`. Tweak both please.

bvaughn commented:
An inline TODO comment is good. You can also add a comment on the version
10 bucket issue in GitHub.

On Tue, Mar 6, 2018 at 2:26 AM Bogdan Chadkin <notifications@github.com>
wrote:

> I think it's only in source/index and source/Table/index. Tweak both
> please.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/bvaughn/react-virtualized/pull/1034#issuecomment-370735667>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABznRjJQ4w2OsP4Ew0SqfsDzdl2Uaitks5tbmRngaJpZM4SbHUe>
> .
>

paulkoegel-wessels commented:
Just updated with the following:
+ in `source/Table/index.js`:
  - renamed the exported function names (from, e.g., `createMultiSort` to `createTableMultiSort`)
  - added aliases for the old function names (`createMultiSort` etc.) for compatibility, added a TODO so they can be removed in version 10
  - renamed the **imported** function names as well - for consistency - their file names (hope this is okay)
+ in `source/index.js`:
  - we no longer need aliases here, only export the new names (`createTableMultiSort` etc.). Didn't add any fallbacks with old names (`createMultiSort` etc.) as there were no such exports before.

Let me know if this works for you.

paulkoegel-wessels commented:
One thing I wasn't sure about:
+ perhaps we should also change type names from `CellDataGetterParams` to `TableCellDataGetterParams`, e.g. here:
https://github.com/paulkoegel-wessels/react-virtualized/blob/9f34739186c679bdeb961ce63cfa601ac2a646dc/source/Table/defaultTableCellDataGetter.js#L2
